README.en-US.md

```diff
@@ -0,0 +1,167 @@
+# Overview
+Markov(MOM Ali functional testing platform)is a new generation of functional testing platform self-developed in the context of test transformation. Compared with traditional functional testing frameworks, it has many advantages, such as visual use case writing management and distributed sandbox environment. and test data construction, test process pipeline management. Besides, many intelligent testing technologies have been derived based on the platform, such as use case recommendation based on Naive Bayes, use case recommendation based on parameter combination expansion filtering, intelligent regression technology base on use case orchestration algorithm, intelligent investigation system based on use case portrait. Accurate and intelligent testing, etc. We can think of Markov as a new generation of the functional testing framework. Compared with the traditional classic testing framework - a.g.: pytest + jenkins - the Markov model has a lower threshold for users, which makes it simple for students who do not understand test development and algorithms.
+The self-service test has achieved the goal of making the world no difficult " to test".
+
+<img src="https://mmbiz.qpic.cn/mmbiz_png/DWQ5ap0dyHMvWOVCYt8M5463BURIjtVD4x8ibtJxYRa0euRLC3ic47oHkRXltOg4SbEtECFwQVoOjlLqIWicpsPwg/640?wx_fmt=png&tp=webp&wxfrom=5&wx_lazy=1&wx_co=1" width="70%" height="70%"></img>
+
+
+# The main function
+
+## `1.pipeline (management)`
+
+- The pipeline is a configuration file, which is mainly related to functional testing. The first is related to the deployment of the test environment, mainly various parameters ( such as IP, deployment scripts, docker and other parameters) and the second is case execution-related parameters (including how to send and verify the module, what data source should be displayed on the page, etc.). It can be abstractly understood that the pipeline only sets the configuration parameters required by a specific module for the deployment page and the debugging page use case. Also, the pipeline is designed as a universal extension form, such as user-defined _contrast test/pressure test/integration test_
+
+<img src="https://wx2.sinaimg.cn/mw690/ac0a3d36ly1gephe1brfij20n80pnnjg.jpg" width="30%" height="50%" ></img>
+
+
+## 2. Use case management
+
+- 可视化的用例管理中，Markov定义了一种面向功能测试的通用页面结构，包含了用例名/描述/业务分组/标签/测试数据/发送query/期望结果等元素，结合pipeline中的测试流程配置，实现了动态渲染用例编辑页的结果，让测试平台能接入更多的测试模块。
```

Comment:
Can this be added:

In the visual use case management, Markov defines a general page structure for functional testing, including elements such as use case name / description / business grouping / tag / test data / send query / expected result, combined with the test process configuration in the pipeline The result of dynamically rendering the use-case edit page is realized, so that the test platform can access more test modules.